### DISCUSSION TITLE
节点位置固定功能

### DISCUSSION THREAD

**Discussion by leozjr** [obsidianzh/forum#64] [Ideas想法]
看关系图的时候，每次关闭重新打开，节点的分布方式都不一样，这对我梳理思路造成了困难 。

能否出一个功能，允许用户**固定关系图中的某些节点**，这样每次打开关系图之后，它的大体框架脑中就还有个印象。

**Comment by Quorafind** [comment]
官方现在还没有对图谱进行动刀，等吧~
